scripts/utils/publishNewVersion.js

```diff
@@ -51,22 +51,18 @@ function publishToNpm(npmTag) {
 }
 
 function releaseDocsVersion(newVersion, previousVersion) {
-  const shouldReplaceWithPreviousVersion = previousVersion && isSameMajorVersion(newVersion, previousVersion);
-  if (shouldReplaceWithPreviousVersion) {
-    removeDocsForVersion(previousVersion);
+  const newDocsVersion = newVersion.split('.', 1).concat('.x');
```

Comment:
should be: `.concat('.x.x')`, no?

Reply:
No? "should" based on what?
Anyway, not in npm semver at least https://github.com/npm/node-semver#x-ranges-12x-1x-12-

Reply:
Oh nice